internal/pkg/cli/env_deploy.go

```diff
@@ -155,6 +158,16 @@ necessary by a service deployment.
 
 In this case, you can run %s to push a modified template, even if there are no immediate changes.
 `, color.HighlightCode("copilot env deploy --force"))
+		}
+		if o.disableRollback {
+			stackName := stack.NameForEnv(o.targetApp.Name, o.targetEnv.Name)
+			rollbackCmd := fmt.Sprintf("aws cloudformation rollback-stack --stack-name %s --role-arn %s", stackName, o.targetEnv.ExecutionRoleARN)
```

Comment:
nit: This is nice so you don't have to change it, I'm just thinking out loud here. 

I'd have expected this log to be in either the (1) `RecommendedActions` method, _or_, (2) we implement a custom error type that implements the `RecommendActions` method.

I don't think (1) fits since we are not exiting `Execute` successfully, so I think (2) is the better route 💭 

So perhaps something like:
```go
err := fmt.Errorf("deploy environment %s: %w", o.name, err)
if o.disableRollback{ 
  return &errWithDisabledRollback{
     parentErr: err,
     stackName: stack.NameForEnv(o.targetApp.Name, o.targetEnv.Name),
     cmd: fmt.Sprintf("aws cloudformation rollback-stack --stack-name %s --role-arn %s", stackName, o.targetEnv.ExecutionRoleARN)
  }
}
return err
```